jmeter/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Stage image (for downloading files etc)
```

Comment:
Can you also please add the license header here?

Reply:
Added the licence header, EOL and changed the standard log file names.